developers/weaviate/model-providers/friendliai/generative.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Generative AI
+sidebar_position: 51
+image: og/docs/integrations/provider_integrations_friendliai.jpg
+# tags: ['model providers', 'friendliai', 'generative', 'rag']
+---
+
+# FriendliAI Generative AI with Weaviate
+
+import BetaPageNote from '../_includes/beta_pages.md';
+
+<BetaPageNote />
+
+:::info Added in `v1.27.0`
+:::
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyConnect from '!!raw-loader!../_includes/provider.connect.py';
+import TSConnect from '!!raw-loader!../_includes/provider.connect.ts';
+import PyCode from '!!raw-loader!../_includes/provider.generative.py';
+import TSCode from '!!raw-loader!../_includes/provider.generative.ts';
+
+Weaviate's integration with FriendliAI's APIs allows you to access open source and their models' capabilities directly from Weaviate.
+
+[Configure a Weaviate collection](#configure-collection) to use an FriendliAI generative AI model, and Weaviate will perform retrieval augmented generation (RAG) using the specified model and your FriendliAI API key.
+
+More specifically, Weaviate will perform a search, retrieve the most relevant objects, and then pass them to the FriendliAI generative model to generate outputs.
+
+![RAG integration illustration](../_includes/integration_friendliai_rag.png)
+
+## Requirements
+
+### Weaviate configuration
+
+Your Weaviate instance must be configured with the FriendliAI generative AI integration (`generative-friendliai`) module.
+
+<details>
+  <summary>For Weaviate Cloud (WCD) users</summary>
```

Comment:
Random question but why is Weaviate Cloud WC**D**?

Reply:
Also interested